eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -38,8 +40,8 @@ do
     matches=`grep -c "${accession}" ${output_folder}/${assembly_accession}.fa`
     if [ $matches -gt 1 ]
     then
-        echo WARNING: Sequence ${genbank_contig} found more than once in the output FASTA file
-        exit_code=1
+        echo Sequence ${genbank_contig} found more than once in the output FASTA file
+        exit_code=2
```

Comment:
so if a sequence is found more than once it overwrites the exit_code=1? we really can't do anything else

Reply:
Yes, any non-zero exit code should be considered a potential error by the top-level automation process and will require manual investigation.